examples/roomevents/src/helpers.js

```diff
@@ -0,0 +1,30 @@
+'use strict';
+
+var Video = require('twilio-video');
+
+/**
+ * Creates a Room and watches for room status events.
+ * @param {string} token - Token for joining the Room
+ * @param {function} onStateChange - Callback for state change.
+ * @returns {CancelablePromise<Room>}
+ */
+function createRoomAndUpdateOnStateChange(token, onStateChange) {
```

Comment:
@makarandp0 You do not need to demonstrate `connect()` here. You can just take in a Room and a state change callback as arguments:
```js
/**
 * Listen to Room reconnection events and update the UI accordingly.
 * @param {Room} room - The Room you have joined
 * @param {function} updateRoomState - Updates the app UI with the new Room state
 * @returns {void}
 */
function setupReconnectionUpdates(room, updateRoomState) {
  room.on('disconnected', (room, error) => {
    if (error.code === 20104) {
      console.log('Signaling reconnection failed due to expired AccessToken!');
    } else if (error.code === 53000) {
      console.log('Signaling reconnection attempts exhausted!');
    } else if (error.code === 53204) {
      console.log('Signaling reconnection took too long!');
    }
    updateRoomState(room.state);
  });
  room.on('reconnected', function() {
    console.log('Reconnected to the Room!');
    updateRoomState(room.state);
  });
  room.on('reconnecting', function(error) {
    if (error.code === 53001) {
      console.log('Reconnecting your signaling connection!', error.message);
    } else if (error.code === 53405) {
      console.log('Reconnecting your media connection!', error.message);
    }
    updateRoomState(room.state);
  });
}
```
?